README.md

```diff
@@ -1,19 +1,23 @@
-# Github actions Sentry app
+# Sentry Github Actions app
```

Comment:
I've renamed the repo because Github does not let me name any app starting with Github.